homework-submission/myapp/src/Components/Data.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Todos from './Todos.json';
+
+class Data extends React.Component {
+
+    state = {
+        Todos
+    }
+
+    isTaskDone = (todo) => {
+        todo.done = !todo.done;
+        this.setState({
+            Todos
+        });
+    }
+
+    renderTodos() {
+        if (this.state.Todos.length === 0) return <p className='noItems'>No items...</p>
+
+        const textStyle = { textDecoration: 'line-through', backgroundColor: '#0455d8' }
```

Comment:
no need to write a function, the contetnt of this function should be inside the render function

Reply:
I have deleted renderTodos function and moved the above code inside the render method.